### ISSUE TITLE
Quick question on loss function

### ISSUE THREAD

**Issue by lovecambi** [HCIILAB/Scene-Text-Removal#19]
Where is the code for the scaling factor in Eq (9)?

In addition, where is the code for the loss function in Eq (2)? It seems that the following line defines the multi-scale loss, but in a different way. 
https://github.com/HCIILAB/Scene-Text-Removal/blob/master/train.py#L142

**Comment by zhangshuaitao** [comment]
@lovecambi , As you said，the multi-scale loss is the following line. 
https://github.com/HCIILAB/Scene-Text-Removal/blob/master/train.py#L142
